### ISSUE TITLE
Access local files for HTML image embedding

### ISSUE THREAD

**Issue by daveholtz** [twosigma/beakerx#708]
It would be excellent to access files that are saved in R or Python (or some other language) in a notebooks HTML cells, but it isn't obvious how to do this right now (if possible at all).

For example, 

```
pdf("a.pdf"); plot(1:10); dev.off()
list.files()
getwd()
```

Creates a file at `/private/tmp/Rserv/conn25723`, but its unclear how to reference this file in an HTML cell as the argument in some img tags, as below:

```
<img src="a.pdf">
```

**Comment by scottdraves** [comment]
related: https://github.com/twosigma/beaker-notebook/issues/430

**Comment by scottdraves** [comment]
released in v1.3
